src/ipc/get-printer-info.ts

```diff
@@ -2,10 +2,21 @@ import { IpcMainInvokeEvent, IpcMain } from 'electron'
 import getConnectedDeviceURIs from '../utils/printing/getConnectedDeviceURIs'
 import printerSchemes from '../utils/printing/printerSchemes'
 import { debug } from '../utils/printing'
+import {
+  getPrinterIppAttributes,
+  PrinterIppAttributes,
+} from '../utils/printing/getPrinterIppAttributes'
 
 export const channel = 'get-printer-info'
 
-export interface PrinterInfo extends Electron.PrinterInfo {
+/**
+ * A collection of info about a printer.
+ * Note that Electron's status is not accurate - its always 3 (idle) even when the printer is stopped.
```

Comment:
It would be pretty easy to test whether this is still true with current Electron. Either way, we should note the highest known version with this bug.

Reply:
I don't think it's actually an Electron bug, but rather part of whatever is responding to requests about the printer's status. In order to get an accurate status, you have to use this secret other IPP URI.